PowerPointLabs/PowerPointLabs/ActionFramework/Content/EffectsLabBlurrinessContentHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.Content
+{
+    [ExportContentRibbonId(
+        TextCollection.EffectsLabBlurrinessFeatureSelected + TextCollection.DynamicMenuId,
+        TextCollection.EffectsLabBlurrinessFeatureRemainder + TextCollection.DynamicMenuId,
+        TextCollection.EffectsLabBlurrinessFeatureBackground + TextCollection.DynamicMenuId)]
+    class EffectsLabBlurrinessContentHandler : ContentHandler
+    {
+        protected override string GetContent(string ribbonId, string ribbonTag)
+        {
+            var feature = ribbonId.Replace(TextCollection.DynamicMenuId, "");
+
+            var xmlString = new System.Text.StringBuilder("<menu xmlns=\"http://schemas.microsoft.com/office/2009/07/customui\">");
+
+            for (int i = 0; i < 7; i++)
+            {
+                xmlString.Append("<button id=\"");
```

Comment:
can make it use String.Format?
and the tag xml can be extracted into constant string for menu, button, menuSeparator, check box, etc